core/src/main/scala/cats/syntax/all.scala

```diff
@@ -122,3 +123,10 @@ trait AllSyntaxBinCompat5 extends ParallelBitraverseSyntax
 trait AllSyntaxBinCompat6 extends ParallelUnorderedTraverseSyntax
 
 trait AllSyntaxBinCompat7 extends SeqSyntax
+
+trait AllSyntaxBinCompat8
```

Comment:
Does this actually need a bincompat group? I'm curious why.

Reply:
I believe this encoding was needed for 2.11, because 2.11 didn't support emitting default method implementations in JVM interfaces. That said, it looks like we dropped 2.11 support, so we probably don't need it.

I can take a look and update it.

Reply:
@armanbilge I've updated this to remove `AllSyntaxBinCompat8`.